include/zephyr/bluetooth/audio/audio.h

```diff
@@ -857,9 +857,9 @@ int bt_audio_codec_cfg_get_freq(const struct bt_audio_codec_cfg *codec_cfg);
  * @param codec_cfg The codec configuration to set data for.
  * @param freq      The assigned numbers frequency to set.
  *
- * @retval The data_len of @p codec_cfg on success
- * @retval -EINVAL if arguments are invalid
- * @retval -ENOMEM if the new value could not set or added due to memory
+ * @retval data_len The data_len of @p codec_cfg on success
```

Comment:
In some other functions where the return value is `len` the description says `length of ...`, maybe the same should be done for `data_len` and `meta_len`, here and in other functions
```suggestion
 * @retval data_len The data length of @p codec_cfg on success
```

Reply:
Hmm, I get your point. `data_len` length here refer to the `data_len` field of `codec_cfg`. Will rewrite a bit

Reply:
+1, this too caught my eye fwiw :)